openedx/core/djangoapps/discussions/forms.py

```diff
@@ -32,3 +32,31 @@ def save(self, commit=True):
     class Meta:
         model = ProgramDiscussionsConfiguration
         fields = '__all__'
+
+
+class ProgramLiveConfigurationForm(forms.ModelForm):
+    """
+    Custom ProgramLiveConfigurationForm form for admin page
+    """
+    pii_share_username = forms.BooleanField(required=False, initial=False)
+    pii_share_email = forms.BooleanField(required=False, initial=False)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if self.instance.lti_configuration:
+            self.fields['pii_share_username'].initial = self.instance.lti_configuration.pii_share_username
+            self.fields['pii_share_email'].initial = self.instance.lti_configuration.pii_share_email
```

Comment:
do you need PII info in live tab?

Reply:
Zoom LTI cannot work without PII sharing